src/components/input/CdrInput.vue

```diff
@@ -117,29 +116,30 @@
     </template>
   </cdr-label-standalone>
 </template>
+
 <script>
-import { defineComponent, useCssModule, computed, ref } from 'vue';
+export default {
+  inheritAttrs: false,
+  customOptions: {}
+}
+</script>
```

Comment:
Interesting, I was wondering when we might run in to needing to do this with the setup syntax